library.json

```diff
@@ -16,7 +16,8 @@
       "matchDepTypes": ["devDependencies"],
       "automerge": true,
       "automergeType": "pr",
-      "automergeStrategy": "squash"
+      "automergeStrategy": "squash",
+      "updateNotScheduled": false
```

Comment:
I'm wondering if we can make a package rule in the base that would make it so that we wouldn't have to apply this to each one. That can come later though since it is kinda nice to see which rules it is applied with while we are testing

Reply:
@prescottprue @meganbailey I'll be interested to see if this works inside packageRules since it's not defined in the schema here:  https://docs.renovatebot.com/renovate-schema.json

Looks like this is applied at the root and can be moved to default.json